plugins/greylisting

```diff
@@ -104,6 +104,13 @@ usable directory from the following list will be used:
 
 =back
 
+=head2 redis_server <host[:port]>
```

Comment:
are we likely to have multiple redis settings? If not, can the setting be named `redis` instead?  I can be convinced otherwise, but we sometimes end up with plugins that have config lines being wrapped b/c they're so long.